library/src/include/auxiliary.hpp

```diff
@@ -170,6 +183,39 @@ constexpr hipblasComputeType_t string_to_hipblas_computetype(const std::string&
         value == "f32_bf16_r" ? HIPBLAS_COMPUTE_32F_FAST_16BF :
         static_cast<hipblasComputeType_t>(0);
 }
+
+HIPBLASLT_EXPORT
+constexpr hipblasComputeType_t string_to_hipblas_computetype_assert(const std::string& value)
+{
+    auto computetytpe = string_to_hipblas_computetype(value);
+    if(static_cast<int>(computetytpe) == 0)
+    {
+        std::cout << "The supported types are f32_r, xf32_r, f64_r, i32_r, f32_f16_r." << std::endl;
+        exit(1);
+    }
+    return computetytpe;
+}
+
+HIPBLASLT_EXPORT
+constexpr hipblasLtEpilogue_t string_to_epilogue_type(const std::string& value)
+{
+    return
+        value == "HIPBLASLT_EPILOGUE_BIAS" ? HIPBLASLT_EPILOGUE_BIAS  :
+        value == "HIPBLASLT_EPILOGUE_DEFAULT" || value == "" ? HIPBLASLT_EPILOGUE_DEFAULT :
+        static_cast<hipblasLtEpilogue_t>(0);
```

Comment:
Better to support all epilogue since this function is in library part but not client part.
once you add it, should consider it from library perspective but not client perspective